lang/cs/Org.Apache.REEF.Wake/Impl/TimerStage.cs

```diff
@@ -15,14 +15,19 @@
 // specific language governing permissions and limitations
 // under the License.
 
-using System.Timers;
+using System;
+using System.Threading;
 
 namespace Org.Apache.REEF.Wake.Impl
 {
     /// <summary>Stage that triggers an event handler periodically</summary>
     public sealed class TimerStage : IStage
     {
-        // private readonly ScheduledExecutorService executor;
+        enum TimeValue : long
```

Comment:
I don't. For some reason I thought the comment said enum, but now that I look back it says constant so I changed it to a public constant.